libcudacxx/test/libcudacxx/cuda/for_each_canceled/for_each_canceled.pass.cpp

```diff
@@ -110,28 +110,11 @@ bool test(int N, F&& f)
     }
 
     f(a, b, c, N, tidx);
-    if (auto e = cudaDeviceSynchronize(); e != cudaSuccess)
-    {
-      std::cerr << "ERROR: synchronize failed" << std::endl;
-      return false;
-    }
 
-    bool success = true;
-    for (int i = 0; i < N; ++i)
-    {
-      if (c[i] != (1 + i))
-      {
-        std::cerr << "ERROR " << i << ", " << c[i] << std::endl;
-        success = false;
-      }
-    }
+    assert(cudaDeviceSynchronize() == cudaSuccess);
```

Comment:
This change dropped the `if (c[i] != (1 + i))` @miscco ?

Reply:
^^ ups